tests/py/unit/test_workflow_utils.py

```
@@ -49,13 +51,24 @@ def _surface_workflow():
     return Workflow.create(config)
 
 
+def _material_stub(number_of_atoms=2, reciprocal_vector_ratios=[1.0, 1.0, 0.5]):
```

Ruff (0.16.1)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)